pages/performances/[id].js

```diff
@@ -0,0 +1,360 @@
+import React, { useState } from 'react'; // React
+import Layout from '@components/Layout'; // Layout wrapper
+import Button from '@components/Button'; // Button Component
+import Dropdown from '@components/Dropdown'; // Dropdown component
+import { getSession } from 'next-auth/client'; // Session handling
+
+import Title from '@components/Title'; // Title
+import TextInput from '@components/Input'; // TextInput Component
+import DancerRedJump from '@assets/dancer-red-jump.svg'; // Jumping Dancer SVG
+import PlayIcon from '@assets/play.svg';
+import styles from '@styles/pages/PerformanceDetails.module.scss';
+
+// Page: Settings
+export default function PerformanceDetails() {
+  const [selectedTab, setSelectedTab] = useState(0);
+  const [showPerformanceSummary, setShowPerformanceSummary] = useState(false);
+  const [feedbackAvailable, setFeedbackAvailable] = useState(true);
+
+  const handleTabClick = () => {
+    setFeedbackAvailable(old => !old); // TODO: Change this!!
```

Comment:
Can't we just do
```javascript
setFeedbackAvailable(!feedbackAvailable)
```

Reply:
Oh yup, good catch. Don't know if it's worth changing though, since this will go away on API hookup. It's just in so that we can toggle b/w the two components (i.e. Empty and JudgeFeedback)